logbook-logstash/src/test/java/org/zalando/logbook/logstash/LogbackLogstashSinkTest.java

```diff
@@ -138,4 +138,64 @@ void shouldLogRequestAndResponse() throws IOException {
         
     }
 
+
+    @Test
+    void shouldLogCorrectJsonForEmptyBody() throws IOException {
```

Comment:
Perhaps this test is not necessary, as the fix is really in JsonHttpLogFormatter.